Ui/Component/Listing/Column/MenuList/PageActions.php

```diff
@@ -112,4 +115,24 @@ private function getDeleteButton(int $menuId): array
             '__disableTmpl' => true,
         ];
     }
+
+    /**
+     * @param int $menuId
+     * @param bool $isActive
+     * @return array
+     */
```

Comment:
I think these docs is a duplicated info and we could remove it (params and return type are already defined within method declaration)

Reply:
Right, removed them.